.github/workflows/release.yml

```
@@ -0,0 +1,38 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "*-v*.*.*"
+
+# We need this to be able to create releases.
+permissions:
+  contents: write
+
+jobs:
+  create-release:
+    name: create-release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+
+      - name: Show the tag
+        run: |
+          echo "tag is: ${{ github.ref_name }}"
```

```suggestion
        run: echo "tag is: ${{ github.ref_name }}"
```
More consistent with the other one-line runs (except if there is a reason I don't know)

---

there is an error when I remove this for some reason